stan/math/rev/mat/fun/cholesky_decompose.hpp

```diff
@@ -223,6 +258,135 @@ class cholesky_scalar : public vari {
     }
   }
 };
+#ifdef STAN_OPENCL
+class cholesky_opencl : public vari {
+ public:
+  int M_;
+  vari** variRefA_;
```

Comment:
We use the naming rules from Google's style guide: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules and https://google.github.io/styleguide/cppguide.html#Variable_Names

In particular, we don't use mixed case for variable names, preferring all lower case with underscores between words.

Reply:
variRefA_ and variRefL_ were replaced. There are similar cases in some other files, should I make a "cleanup mixed case" issue?

Reply:
Ah that'd be great, thanks! In retrospect I should have mentioned this earlier probably, but I felt like we already had a lot to deal with that was more important. Now things are all really good and so we're getting into marginal returns 😂